plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -143,6 +145,12 @@ func (c *nodePlugin) Admit(a admission.Attributes) error {
 		}
 		return admission.NewForbidden(a, fmt.Errorf("disabled by feature gate %s", features.NodeLease))
 
+	case csiNodeInfoResource:
+		if c.features.Enabled(features.CSICRDAutoInstall) {
```

Comment:
Is `CSICRDAutoInstall` the right feature to check for this? This shouldn't really care how the CRD is installed, just if the node is using it